### PR TITLE
Replace \newcommand in the documetation; fix the doc in read the doc

### PR DESCRIPTION
**Pull request by bouhlelma**

### CODE REVIEW

File: doc/_src_docs/methods/idw.rst [hunk 1]
```diff
@@ -40,6 +34,7 @@ where :math:`p` a positive real number, called the power parameter.
 This parameter must be strictly greater than 1 for the derivatives to be continuous.
 
 .. [1] Shepard, D., A Two-dimensional Interpolation Function for Irregularly-spaced Data, Proceedings of the 1968 23rd ACM National Conference, 1968, pp. 517--524.
+.. [1] Shepard, D., A Two-dimensional Interpolation Function for Irregularly-spaced Data, Proceedings of the 1968 23rd ACM National Conference, 1968, pp. 517--524.
```
Comment (hwangjt) [inline doc/_src_docs/methods/idw.rst]: Looks like this was unintentionally copied twice

File: doc/_src_docs/methods/idw.rstx [hunk 2]
```diff
@@ -40,6 +34,7 @@ where :math:`p` a positive real number, called the power parameter.
 This parameter must be strictly greater than 1 for the derivatives to be continuous.
 
 .. [1] Shepard, D., A Two-dimensional Interpolation Function for Irregularly-spaced Data, Proceedings of the 1968 23rd ACM National Conference, 1968, pp. 517--524.
+.. [1] Shepard, D., A Two-dimensional Interpolation Function for Irregularly-spaced Data, Proceedings of the 1968 23rd ACM National Conference, 1968, pp. 517--524.
```
Comment (hwangjt) [inline doc/_src_docs/methods/idw.rstx]: Looks like this was unintentionally copied twice

Comment (hwangjt) [inline doc/_src_docs/methods/idw.rstx]: Looks like this was unintentionally copied twice

File: doc/_src_docs/methods/idw.rst (hunk 1 above)
Comment (bouhlelma) [reply doc/_src_docs/methods/idw.rst]: corrected

**Pull request merged**